test/integration/edition_edit_test.rb

```diff
@@ -108,53 +94,149 @@ class EditionEditTest < IntegrationTest
+          assert page.has_css?(".govuk-input", count: 1)
+        end
+
+        should "navigate to 'confirm-unpublish' page when clicked on 'Continue' button" do
+          click_button("Continue")
+          assert_equal(page.current_path, "/editions/#{@edition.id}/unpublish/confirm-unpublish")
+        end
       end
 
-      should "navigate to 'confirm-unpublish' page when clicked on 'Continue' button" do
-        click_button("Continue")
-        assert_equal(page.current_path, "/editions/#{@edition.id}/unpublish/confirm-unpublish")
+      context "when state is not publish" do
+        setup do
+          edition = FactoryBot.create(:guide_edition, state: "draft")
+          visit edition_path(edition)
+        end
+
+        should "not show unpublish tab" do
+          assert page.has_no_text?("Unpublish")
+        end
       end
     end
+  end
 
-    context "confirm unpublish" do
+  context "admin tab" do
+    context "do not have required permissions" do
       setup do
-        click_link("Unpublish")
-        click_button("Continue")
+        user = FactoryBot.create(:user, name: "Stub User")
+        login_as(user)
+        edition = FactoryBot.create(:guide_edition, state: "draft")
+        visit edition_path(edition)
       end
 
-      should "show 'Unpublish' header and document title" do
-        assert page.has_text?("Unpublish")
-        assert page.has_text?(@edition.title.to_s)
+      should "not show when user is not govuk editor or welsh editor" do
+        assert page.has_no_text?("Admin")
       end
 
-      should "show 'cannot be undone' banner" do
-        assert page.has_text?("If you unpublish a page from GOV.UK it cannot be undone.")
+      should "not show when user is welsh editor and edition is not welsh" do
+        login_as(FactoryBot.create(:user, :welsh_editor, name: "Stub User"))
+        edition = FactoryBot.create(:guide_edition, state: "draft")
+        visit edition_path(edition)
+
+        assert page.has_no_text?("Admin")
       end
+    end
 
-      should "show 'Unpublish document' button and 'Cancel' link" do
-        assert page.has_button?("Unpublish document")
-        assert page.has_link?("Cancel")
+    context "has required permissions" do
+      setup do
+        login_as(FactoryBot.create(:user, :govuk_editor, name: "Stub User"))
+        edition = FactoryBot.create(:guide_edition, title: "Edit page title", state: "draft")
+        visit edition_path(edition)
+        click_link("Admin")
+      end
+
+      context "when state is not 'fact_check'" do
+        should "show 'Admin' header and not show 'Skip fact check' button" do
+          within :css, ".gem-c-heading" do
+            assert page.has_text?("Admin")
+          end
+          assert page.has_no_button?("Skip fact check")
+        end
+      end
+
+      context "when state is 'fact_check'" do
+        setup do
+          @edition = FactoryBot.create(:guide_edition, state: "fact_check")
```

Comment:
We should probably push the edition creation down into the "when state is not 'fact_check'" context, to match the "when state is 'fact_check'" context.

Reply:
have now, updated the file and made a few improvements which should also improve this issue